README.md

```diff
@@ -8,6 +8,83 @@
 
 # Verifiable Credential Transparency
 
+The Verifiable Credential Transparency (VCT) is based on certificate transparency [RFC6962](https://datatracker.ietf.org/doc/html/rfc6962).
+The Credentials are included into VCT as append-only [Merkle Tree](https://en.wikipedia.org/wiki/Merkle_tree) logs.
+VCT uses the existing Merkle Tree implementation provided by [Trillian](https://github.com/google/trillian).
+VCT was created for the [orb](https://github.com/trustbloc/orb#orb-did-method) project and its usage is described [here](https://trustbloc.github.io/did-method-orb/#witness-ledger).
+
+## Build and test
+
+To build and test VCT you need:
+
+* Go 1.16 or later
+* GNU Make
+* Bash
+* Docker and Docker-Compose
+
+Run tests:
+
+```make unit-test bdd-test```
+
+## Deploy
+
+VCT itself is a compound of three essential components.
+
+* log-server
+* log-signer
+* vct
+
+To deploy the service you would need to build them first.
+
+`make build-vct build-log-server build-log-signer`
+
+1. Run the log-server.
+
+Log server depends on the database. To run DB you can simply use `docker-compose` for that.
+
+NOTE: We support MySQL and Postgres. If you run your own DB service, do not forget to import schema.
+See [MySQL schema](https://github.com/trustbloc/vct/blob/main/test/bdd/fixtures/vct/mysql-config/mysql_config.sql)
+and [Postgres schema](https://github.com/trustbloc/vct/blob/main/test/bdd/fixtures/vct/postgres-config/postgres_config.sql).
+
+Run MySQL by using `docker-compose`:
+
+`cd ./test/bdd/fixtures/vct && docker-compose up vct.mysql`
+
+Then run the log-server.
+
+`./build/bin/log-server --mysql_uri="root@tcp(localhost:3306)/test"`
+
+Log server will start the RPC server on `localhost:8090` and  HTTP server on `localhost:8091`.
+
+Use help flag to find out all available flags `./build/bin/log-server -h`.
+
+2. Run the log-signer.
+
+Log signer must be run with the same DB configuration as a `log-server`.
+Log signer will start the RPC server on `localhost:8090` and  HTTP server on `localhost:8091`.
+Since those ports are already occupied by the `log-server`, let's change them.
+
+` ./build/bin/log-signer --mysql_uri="root@tcp(localhost:3306)/test" --force_master=true --http_endpoint=0.0.0.0:8099 --rpc_endpoint=0.0.0.0:8098   `
+
+Use help flag to find out all available flags `./build/bin/log-signer -h`.
+
+3.  Run the vct.
+
+` ./build/bin/vct start --logs=maple2021:rw@127.0.0.1:8090`
+
+The required argument here is a `logs`. It specifies an alias for the log, permissions and `log-service` RPC endpoint.
```

Comment:
The required argument here is `--logs`.

(To be consistent with the --dsn arg below.)